.circleci/config.yml

```diff
@@ -26,19 +26,16 @@ jobs:
       - run:
           name: "Package debs (debuild)"
           command: |
-            # build is going to fail but it's needed to generate the debs
-            ls -lah
-            pwd
             echo "$PWD"
-            GITHUB_WORKSPACE="$PWD" greenzie-dkms mkdeb -d
-            cd /tmp/debians/ && tar cvf /tmp/debians/alldebs.tar *.deb
+            ls -lah
+            bash -x "$PWD/build_dmks.sh"    
```

Comment:
New build process with a script that can be reade3d to understand how the process works.